Documentation/Background Indexing.md

```diff
@@ -0,0 +1,42 @@
+# Background Indexing
+
+Background indexing in SourceKit-LSP is available as an experimental feature. This guide shows how to set up background indexing and which caveats to expect.
+
+> [!WARNING]
+> Background indexing is still an experimental feature and still has some non-trivial limitations and bugs (see the known issues).
+
+## Set Up
+
+Background indexing is still under active, rapid development, so please build SourceKit-LSP from the `main` branch using the following commands. Things are changing rapidly, so rebuilding daily is not a bad practice.
+
+```bash
+$ git clone sourcekit-lsp
+$ swift package update
+$ swift build -c release
+```
+
+Next, point your editor to use the just-built copy of SourceKit-LSP and enable background indexing by passing `--experimental-feature background-indexing` to sourcekit-lsp. In VS Code, this can be done by adding the following to your settings.json
+```json
+"swift.sourcekit-lsp.serverPath": "/path/to/sourcekit-lsp/.build/arm64-apple-macosx/release/sourcekit-lsp",
```

Comment:
Could use the `.build/release` symlink to avoid including the target in the path here

Reply:
Good idea 👍🏽